weed/server/volume_grpc_vacuum.go

```diff
@@ -29,11 +34,16 @@ func (vs *VolumeServer) VacuumVolumeCompact(req *volume_server_pb.VacuumVolumeCo
 	resp := &volume_server_pb.VacuumVolumeCompactResponse{}
 	reportInterval := int64(1024 * 1024 * 128)
 	nextReportTarget := reportInterval
-
+	fs, fsErr := procfs.NewDefaultFS()
 	var sendErr error
 	err := vs.store.CompactVolume(needle.VolumeId(req.VolumeId), req.Preallocate, vs.compactionBytePerSecond, func(processed int64) bool {
 		if processed > nextReportTarget {
 			resp.ProcessedBytes = processed
+			if fsErr == nil && numCPU > 0 {
+				if fsLa, err := fs.LoadAvg(); err == nil {
+					resp.PercentLoadAvg_1M = uint32(math.Round(fsLa.Load1 * 100 / float64(numCPU)))
```

Comment:
remove percent, rounding and multiply.

Reply:
It made sense to take utilization as a percentage, so that there would be no confusion with the original LA